nasl/nasl_ssh.c

```diff
@@ -1064,7 +1064,8 @@ nasl_ssh_login_interactive (lex_ctxt *lexic)
     {
       /* Our strategy for kbint is to send the password to the first
          prompt marked as non-echo.  */
-      while ((rc = ssh_userauth_kbdint (session, NULL, NULL)) == SSH_AUTH_INFO)
+      rc = ssh_userauth_kbdint (session, NULL, NULL);
+      while (rc == SSH_AUTH_INFO)
```

Comment:
```suggestion
      while (ssh_userauth_kbdint (session, NULL, NULL) == SSH_AUTH_INFO)
```
We actually never use the value of `rc`, except for the while condition. I guess we can remove `rc` completely and put the function directly into the while loop